src/style.css

```
@@ -0,0 +1,103 @@
+body {
+  margin: 0%;
```

```suggestion
  margin: 0;
```
для нуля процент не обов'язковий